product-service/test/handlers/get-products-list.test.js

```diff
@@ -1,39 +1,57 @@
 import { getProductsList } from '../../src/handlers/get-products-list';
-
 import * as productRepository from '../../src/repositories/product-repository';
 import * as response from '../../src/services/response';
 
-import products from '../../resources/product-list.json';
+import { productMock } from '../mocks/product.mock';
+import { responseMock } from '../mocks/response.mock';
 
-global.console = { log: jest.fn() };
+const products = [
+  productMock(),
+  productMock(),
+  productMock(),
+]
 
 beforeEach(() => {
   productRepository.findProducts = jest.fn();
   response.ok = jest.fn();
   response.serverError = jest.fn();
+  global.console = {
+    info: jest.fn(),
+    error: jest.fn(),
+  };
 });
 
 afterEach(() => {
   productRepository.findProducts.mockClear();
   response.ok.mockClear();
   response.serverError.mockClear();
+  global.console.info.mockClear()
+  global.console.error.mockClear();
 });
 
 it('Should invoke functions with right params', async () => {
+  const serverOkResponse = responseMock(200, JSON.stringify(products));
   productRepository.findProducts.mockResolvedValue(products);
+  response.ok.mockReturnValue(serverOkResponse);
 
-  await getProductsList();
+  const res = await getProductsList();
 
+  expect(global.console.info).toHaveBeenCalled();
   expect(productRepository.findProducts).toHaveBeenCalled();
   expect(response.ok).toHaveBeenCalled();
   expect(response.ok.mock.calls[0][0]).toStrictEqual(products);
```

Comment:
really strange `response.ok.mock.calls[0][0]`

use this method `expect(response.ok).nthCalledWith(1, products)` more strict and seems it that u need.